### DISCUSSION TITLE
Autohide on single window on workspace

### DISCUSSION THREAD

**Discussion by redoxahmii** [Jas-SinghFSU/HyprPanel#553] [Q&A] [answered]
I am not currently using Hyprpanel and didn't really find anything in the docs for autohide.
Is it possible for it autohide if there is a single window on a workspace and if you open another window it would appear again similar to how KDE plasma handles it.

**Comment by Jas-SinghFSU** [comment]
No but it's coming with the AGSv2/Astal update.

**Reply by redoxahmii** [reply]
I'll wait then when it updates to AGSv2.
Thanks for the quick reply.

**Reply by Jas-SinghFSU** [reply, marked as answer]
This has been implemented now. You can find the option in `Configuration > Bar > Auto Hide`.
